### PR TITLE
Update spring cloud reademe

### PR DESCRIPTION
**Pull request by yiliuTo**
enrich / fullfill READMEs of spring cloud modules:
- spring cloud autoconfiguration
- spring cloud context
- spring cloud messaging
- spring cloud starter for app configuration
- spring cloud starter cache
- spring cloud stream binder test
- spring cloud telemetry
- spring integration core
- spring integration test
- spring cloud storage
- spring cloud storage starter
- spring cloud storage queue integration

### CODE REVIEW

File: sdk/spring/azure-spring-cloud-context/README.md [hunk 1]
```diff
@@ -1,9 +1,61 @@
...
 
-## Key concepts
 ## Getting started
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven][maven] 3.0 and above
+
+### Include the package
+[//]: # "{x-version-update-start;com.microsoft.azure:spring-cloud-azure-context;current}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-context/README.md:13]: Is the double-quote added accidentally?

File: sdk/spring/azure-spring-cloud-context/README.md [hunk 2]
```diff
@@ -1,9 +1,61 @@
...
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>spring-cloud-azure-context</artifactId>
+    <version>1.2.8-beta.1</version>
+</dependency>
+```
+[//]: # "{x-version-update-end}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-context/README.md:21]: same here.

File: sdk/spring/azure-spring-cloud-context/README.md [hunk 3]
```diff
@@ -1,9 +1,61 @@
...
+
 ## Key concepts
+Azure contexts are PowerShell objects representing your active subscription to run commands against, and the authentication information needed to connect to an Azure cloud. With Azure contexts, Azure PowerShell doesn't need to reauthenticate your account each time you switch subscriptions.
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-context/README.md:24]: I don't think our azure-context is the PowerShell object 😄. But I believe they share the same abstraction.

File: sdk/spring/azure-spring-cloud-starter-appconfiguration-config/README.md [hunk 4]
```diff
@@ -1,54 +1,46 @@
...
-Please use this `sample` as a reference for how to use this starter.
-
-### Dependency Management
-
+### Include the package
 There are two libraries that can be used spring-cloud-azure-appconfiguration-config and spring-cloud-azure-appconfiguration-config-web. There are two differences between them the first being the web version takes on spring-web as a dependency, and the web version will attempt a refresh when the application is active when the cache expires. For more information on refresh see the [Configuration Refresh](#Configuration-Refresh) section.
 
-#### Maven Coordinates
-
+[//]: # "{x-version-update-start;com.microsoft.azure:spring-cloud-azure-appconfiguration-config;current}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-appconfiguration-config/README.md:16]: double-quote here.

File: sdk/spring/azure-spring-cloud-starter-appconfiguration-config/README.md [hunk 5]
```diff
@@ -1,54 +1,46 @@
...
 ```xml
 <dependency>
     <groupId>com.microsoft.azure</groupId>
     <artifactId>spring-cloud-azure-appconfiguration-config</artifactId>
-    <version>{version}</version>
+    <version>1.2.8-beta.1</version>
 </dependency>
 ```
+[//]: # "{x-version-update-end}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-appconfiguration-config/README.md:24]: same here

File: sdk/spring/azure-spring-cloud-starter-cache/README.md [hunk 6]
```diff
@@ -1,8 +1,66 @@
...
 
-## Key concepts
 ## Getting started
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven][maven] 3.0 and above
+
+### Include the package
+[//]: # "{x-version-update-start;com.microsoft.azure:spring-starter-azure-cache;current}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-cache/README.md:13]: same here.

File: sdk/spring/azure-spring-cloud-starter-cache/README.md [hunk 7]
```diff
@@ -1,8 +1,66 @@
...
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>spring-starter-azure-cache</artifactId>
+    <version>1.2.8-beta.1</version>
+</dependency>
+```
+[//]: # "{x-version-update-end}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-cache/README.md:21]: same here.

File: sdk/spring/azure-spring-cloud-starter-storage/README.md [hunk 8]
```diff
@@ -1,38 +1,33 @@
...
-## Key concepts
 ## Getting started
-## Examples
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven][maven] 3.0 and above
+
+### Include the package
+[//]: # "{x-version-update-start;com.microsoft.azure:spring-starter-azure-storage;current}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-storage/README.md:12]: same here.

File: sdk/spring/azure-spring-cloud-starter-storage/README.md [hunk 9]
```diff
@@ -1,38 +1,33 @@
...
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>spring-starter-azure-storage</artifactId>
+    <version>1.2.8-beta.1</version>
+</dependency>
+```
+[//]: # "{x-version-update-end}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-storage/README.md:20]: same here.

File: sdk/spring/azure-spring-cloud-storage/README.md [hunk 10]
```diff
@@ -1,9 +1,65 @@
...
 
-## Key concepts
 ## Getting started
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven][maven] 3.0 and above
+
+### Include the package
+[//]: # "{x-version-update-start;com.microsoft.azure:spring-cloud-azure-storage;current}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-storage/README.md:13]: same here.

File: sdk/spring/azure-spring-cloud-storage/README.md [hunk 11]
```diff
@@ -1,9 +1,65 @@
...
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>spring-cloud-azure-storage</artifactId>
+    <version>1.2.8-beta.1</version>
+</dependency>
+```
+[//]: # "{x-version-update-end}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-storage/README.md:21]: same here.

File: sdk/spring/azure-spring-integration-storage-queue/README.md [hunk 12]
```diff
@@ -1,9 +1,67 @@
...
 
-## Key concepts
 ## Getting started
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven][maven] 3.0 and above
+
+### Include the package
+[//]: # "{x-version-update-start;com.microsoft.azure:spring-integration-storage-queue;current}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-storage-queue/README.md:13]: same here.

File: sdk/spring/azure-spring-integration-storage-queue/README.md [hunk 13]
```diff
@@ -1,9 +1,67 @@
...
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>spring-integration-storage-queue</artifactId>
+    <version>1.2.8-beta.1</version>
+</dependency>
+```
+[//]: # "{x-version-update-end}"
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-storage-queue/README.md:21]: same here.

File: sdk/spring/azure-spring-integration-storage-queue/README.md [hunk 14]
```diff
@@ -1,9 +1,67 @@
...
+[src]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-integration-storage-queue
+[package]: https://mvnrepository.com/artifact/com.microsoft.azure/spring-integration-storage-queue
+[refdocs]: https://azure.github.io/azure-sdk-for-java/spring.html#spring-integration-storage-queue
+[sample]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-storage-queue-integration-sample
+[logging]: https://github.com/Azure/azure-sdk-for-java/wiki/Logging-with-Azure-SDK#use-logback-logging-framework-in-a-spring-boot-application
+[azure_subscription]: https://azure.microsoft.com/free
+[logging_doc]: https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-logging
+[contributing_md]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/CONTRIBUTING.md
+[maven]: http://maven.apache.org/
+[source_code]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-integration-eventhubs
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-storage-queue/README.md:67]: This line is not used and should be removed.

File: sdk/spring/azure-spring-cloud-messaging/README.md [hunk 15]
```diff
@@ -1,6 +1,6 @@
 # Azure Spring Cloud Messaging client library for Java
+This package is an internal package providing supporting config and annotation classes for Azure Messaging services.
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-messaging/README.md:2]: Is this package only used internally?

**Review by saragluna — CHANGES_REQUESTED** [review]
Thanks so much for completing these docs.

File: sdk/spring/azure-spring-cloud-context/README.md (hunk 3 above)
Comment (yiliuTo) [reply sdk/spring/azure-spring-cloud-context/README.md]: I updated this part, please help to check, thx

**Comment by yiliuTo** [comment]
Close this PR because accidentally rebase from master

**Pull request closed without merging**